pkg/controller/history/controller_history.go

```diff
@@ -56,20 +56,17 @@ func ControllerRevisionName(prefix string, hash uint32) string {
 }
 
 // NewControllerRevision returns a ControllerRevision with a ControllerRef pointing to parent and indicating that
-// parent is of parentKind. The ControllerRevision has labels matching selector, contains Data equal to data, and
+// parent is of parentKind. The ControllerRevision has labels matching pod, contains Data equal to data, and
```

Comment:
CR's labels is a copy of template labels. Matching is between selector and labels.